src/bindings/jna/libelektra5j/src/test/java/org/libelektra/GOptsTest.java

```diff
@@ -1,13 +1,11 @@
 package org.libelektra;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
```

Comment:
We should probably add some kind of automatic check for things like this. Otherwise one PR expands the `.*` imports and the next one collapses them again. Do you have plans to add that to your future Gradle setup?

Reply:
Not yet. I will not it down and eventually create an issue as soon as the gradle build is merged.